### PR TITLE
Update keyboards piece to provide access to code within the article

### PR DESCRIPTION
**Pull request by rlskoeser**
We've talked about startwords pieces including code snippets, but the one piece we've published so far that was technical enough to do so, we don't show the code! I think we should make the code more easily available in the article, since people will be much more likely to look at it there (vs. at least two clicks otherwise).

in this pr:
- new short code (& accompanying styles) for displaying code from a file in a collapsed details/summary element
- document the short code in theme readme
- use the new short code to embed a viewable version of the source code in the keyboards article for each embedded keyboard

----- 

to review:
- [x] review the file changes in this pr
- [x] look at the article with code embedded in the piece on render site: https://startwords-dev-pr-300.onrender.com/issues/2/strangers-in-the-landscape/

### CODE REVIEW

File: themes/startwords/README.md [hunk 1]
```diff
@@ -262,6 +262,18 @@ Either `pdf-img` or `youtube_id` should be specified.
...
+
+Use to display highlighted source code from a file in a details/summary element.
+Path should be relative to content directory.
+
+Example use:
+```
+{{< source_code source_file="path/to/code.html" format="html" >}}
+```
+
+[view source](layouts/shortcodes/source_code.html)
```
Comment (gwijthoff) [inline themes/startwords/README.md:275]: @rlskoeser Is it just me, or could the "view source" link right beneath the `source_code` example use be confusing? At first glance, I still thought the "view source" hyperlink was _part_ of the "Example use." But again, that might just be me, looking at one shortcode documentation in isolation. Someone reading the entire theme README would probably recognize that all of the shortcodes contain a "view source" link at the bottom of each section.

Comment (gwijthoff) [reply themes/startwords/README.md:275]: Only thing I can think of is to change the "view source" hyperlink at the end of each shortcode section in the theme README to something like "view this shortcode", or "view shortcode source". Shortcode sourcecode? 🤭 Maybe this is a nonissue.

Comment (rlskoeser) [reply themes/startwords/README.md:275]: @gwijthoff OMG it _is_ kind of confusing! I didn't even think about that, just followed the pattern from how we documented the shortcodes.  (oh no, could be "view source code shortcode source code " 🤣 ) 

I think isn't quite as bad on the [rendered version of the readme](https://github.com/Princeton-CDH/startwords/tree/chore/include-keyboards-code/themes/startwords) as it is in the markdown. I'm open to revising the label, but if we do we may want to revise the others as well.

Comment (gwijthoff) [reply themes/startwords/README.md:275]: @rlskoeser agreed. Let's leave it as is for now. We can always revisit later.

**Pull request merged**